<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Estabelecimento</class>
 <widget class="QDialog" name="Estabelecimento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loja</string>
  </property>
  <widget class="QPushButton" name="pushButton_salvar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancelar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>479</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Endereço</string>
    </attribute>
    <widget class="QLabel" name="label_estado">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>UF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Fone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Número:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomeloja">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_cidades">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_procurarua">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cidade">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cidade</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_numero">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_estados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Cidade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_procurar">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bairro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bairro</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_fone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>341</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomerua">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cep">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CEP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>479</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Dados Contábeis</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_cnpj">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inscrição Estadual</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inscrest">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CNPJ</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>479</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Redes Sociais</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Site</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_site">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_facebook">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FaceBook</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>479</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Regras</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Cobrança de Entrega</string>
     </property>
     <widget class="QRadioButton" name="radioButton_entregavalor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor R$</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_entregaperc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Percentual %</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Multiplas Seleções</string>
     </property>
     <widget class="QRadioButton" name="radioButton_mult_maior">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Maior valor</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_mult_media">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Média</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
